fdbserver/fdbserver.actor.cpp

```diff
@@ -1493,7 +1493,7 @@ int main(int argc, char* argv[]) {
 					if (listenError.isReady()) listenError.get();
 				} catch (Error& e) {
 					TraceEvent("BindError").error(e);
-					fprintf(stderr, "Error initializing networking with public address %s and listen address %s\n", publicAddress.toString().c_str(), listenAddress.toString().c_str());
+					fprintf(stderr, "Error initializing networking with public address %s and listen address %s (%s)\n", publicAddress.toString().c_str(), listenAddress.toString().c_str(), e.what());
```

Comment:
Does this error information already get propagated into the trace event by setting it as the error?

Reply:
Yes, but I had at one point run in a situation where it was easier for me to get the output of the process than the trace logs. It seemed reasonable to me that if we're going to print an error here, that we provide a reasonable description of the error as well.

Reply:
Oh, my question was more of the form, "Do we need to also add that to the TraceEvent?" rather than, "Why would we add it to the stderr message?", but I see that it's already there, so that's fine.